meta-resin-common/recipes-containers/balena/balena_git.bb

```diff
@@ -119,20 +119,28 @@ do_compile() {
 
 do_install() {
 	mkdir -p ${D}/${bindir}
-	install -m 0755 ${S}/src/import/bundles/dynbinary-balena/balena ${D}/${bindir}/balena
+	install -m 0755 ${S}/src/import/bundles/dynbinary-balena/balena-engine ${D}/${bindir}/balena-engine
 	install -d ${D}/boot
 	install -m 0755 ${S}/src/import/cmd/mobynit/mobynit ${D}/boot/init
 	echo ${BALENA_STORAGE} > ${D}/boot/storage-driver
 
-	ln -sf balena ${D}/${bindir}/balenad
-	ln -sf balena ${D}/${bindir}/balena-containerd
-	ln -sf balena ${D}/${bindir}/balena-containerd-shim
-	ln -sf balena ${D}/${bindir}/balena-containerd-ctr
-	ln -sf balena ${D}/${bindir}/balena-runc
-	ln -sf balena ${D}/${bindir}/balena-proxy
+	ln -sf balena-engine ${D}/${bindir}/balena
+	ln -sf balena-engine-daemon ${D}/${bindir}/balenad
+	ln -sf balena-engine-containerd ${D}/${bindir}/balena-containerd
+	ln -sf balena-engine-containerd-shim ${D}/${bindir}/balena-containerd-shim
+	ln -sf balena-engine-containerd-ctr ${D}/${bindir}/balena-containerd-ctr
+	ln -sf balena-engine-runc ${D}/${bindir}/balena-runc
+	ln -sf balena-engine-proxy ${D}/${bindir}/balena-proxy
```

Comment:
these links need to all point to balena-engine now that https://github.com/resin-os/balena-engine/pull/113 is in